disco_aws_automation/disco_elb.py

```diff
@@ -242,8 +250,20 @@ def get_or_create_elb(self, hostclass, security_groups, subnets, hosted_zone_nam
 
         self.route53.create_record(hosted_zone_name, cname, 'CNAME', elb['DNSName'])
 
-        self._setup_health_check(elb_id, health_check_url, instance_protocol, instance_port, elb_name)
-        self._setup_sticky_cookies(elb_id, elb_ports, sticky_app_cookie, elb_name)
+        for mapping in port_config.port_mappings:
+            self._setup_health_check(
+                elb_id,
+                health_check_url,
```

Comment:
I assumed that calling _setup_health_check multiple times would create multiple health checks.  So it _should_ be creating a health check for each port mapping (I'm pretty sure this is what I saw on our ELB, but I'll double check).

If that's the case, then the main issue here is that it uses the same health check URL for all HTTP(S) listeners, which I can fix easily enough.